server/routes/challenges/create/index.ts

```diff
@@ -19,8 +21,7 @@ const createChallengeRoutes = (services: Services): Router => {
   const detailController = new DetailController(challengeService)
 
   router.use('/', [
-    // TODO - enable this line when we understand the RBAC roles and permissions
-    // checkUserHasPermissionTo(ApplicationAction.RECORD_CHALLENGES),
+    checkUserHasPermissionTo(ApplicationAction.RECORD_CHALLENGES),
```

Comment:
This is how the middleware that checks user permissions is wired into the routes